common/src/main/java/com/wynntils/services/statistics/StatisticsCollectors.java

```diff
@@ -45,4 +49,19 @@ public void onLootrunFailed(LootrunFinishedEvent.Failed event) {
                 StatisticKind.LOOTRUNS_CHALLENGES_COMPLETED, event.getChallengesCompleted());
         Services.Statistics.addToStatistics(StatisticKind.LOOTRUNS_TIME_ELAPSED, event.getTimeElapsed());
     }
+
+    @SubscribeEvent
+    public void onMythicFoundEvent(MythicFoundEvent event) {
+        Services.Statistics.increaseStatistics(StatisticKind.MYTHICS_FOUND);
+
+        Services.Statistics.addToStatistics(
+                StatisticKind.LOOTRUNS_CHALLENGES_WITHOUT_MYTHIC, Models.Lootrun.dryPulls.get());
```

Comment:
This should have a check to make sure the mythic was found in a lootrun and not outside one

Reply:
The event is only fired on lootrun/lootchest finds, but yes, we should check for lootrun.